src/QuestionViewer/index.js

```diff
@@ -10,13 +9,16 @@ class QuestionViewer extends Component {
   }
   async componentDidMount() {
     const { qno } = this.props.params;
-    var res = await fetchWithAuth(`/questions/${qno}`);
+    var res = await window.fetchWithAuth(`/questions/${qno}`);
     res = await res.json();
     if (!res.error) this.setState({ question: res, loading: false })
     else this.setState({ error: res.error, loading: false })
   }
   render() {
     const { loading, question, error } = this.state
+    const quesno = parseInt(this.props.params.qno,10)
+    const prev = `/question/${quesno-1}`
+    const nxt = `/question/${quesno+1}`
```

Comment:
No need to define as you will use them only once if you code it like I suggested. However your choice.